src/content/article/ubuntu-team-kodi-ppa-officially-retired/index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Ubuntu Team-Kodi PPA Officially Retired
+date: 2024-03-24T12:51:12.357Z
+author: Team Kodi
+tags:
+  - Community
+  - Developer
+featured_image:
+  src: /images/blog/ubuntu.webp
+  title: Ubuntu logo
+  alt: Ubuntu Linux logo
+---
+The Team Kodi PPA has long been a staple for Ubuntu (and similar) users wishing to use more recent, and less adulterated, versions of Kodi.
+
+It is with sadness that the team has come to a decision to officially retire the PPA due to the overhead of maintaining it.
+
+We want to express our sincere thanks to **wsnipex** for maintaining it for so many years, tirelessly making it available to countless users in the endeavour of making Kodi more easily accessible for the masses on many of the more popular Linux distributions.
+
+The Linux world is shifting slowly to containers, and so, going forward, the team is looking to utilise Flatpak to hopefully give Linux users a similar, but more maintainable, "pure" Kodi application. We believe this will also allow us to provide greater accessibility from more distributions than the PPA was ever able to.
```

Comment:
I think "greater access" is better than "greater accessibility" in this context. Or maybe "provide greater coverage of distributions".

Reply:
Agree